cmd/docs.go

```diff
@@ -37,18 +39,33 @@ var docsCmd = &cobra.Command{
 				u.LogErrorAndExit(schema.CliConfiguration{}, err)
 			}
 
+			// Detect terminal width if not specified in `atmos.yaml`
+			maxWidth := cliConfig.Settings.Docs.MaxWidth
+			if maxWidth == 0 {
+				if term.IsTerminal(int(os.Stdout.Fd())) {
+					w, _, err := term.GetSize(int(os.Stdout.Fd()))
+					if err == nil {
+						maxWidth = int(w)
+					}
+
+					if maxWidth > 120 {
+						maxWidth = 120
+					}
+
+					if maxWidth == 0 {
+						maxWidth = 80
+					}
```

Comment:
This will need to move higher, if this other suggestion is accepted

https://github.com/cloudposse/atmos/pull/757/files#r1828219437